packages/tempus/src/Tempus_StepperRKButcherTableau.hpp

```diff
@@ -3179,7 +3319,7 @@ class StepperDIRK_General :
 
   StepperDIRK_General(
     const Teuchos::RCP<const Thyra::ModelEvaluator<Scalar> >& appModel,
-    const Teuchos::RCP<StepperExplicitRKObserverComposite<Scalar> >& obs,
+    const Teuchos::RCP<StepperDIRKObserver<Scalar> >& obs,
```

Comment:
I like the switch away from the Composite here!

Reply:
Would you prefer Steppers have an Observer instead of a composite observer?  I think @sconde would prefer the Stepper have a composite observer.  Just to get my preference out there is I would prefer the Steppers have an Observer, and a composite observer could set on the Stepper's observer if needed.

Reply:
I would prefer the abstract base class be set by the user. If they want multiple observers they can create the composite externally and nest multiples in the ordering that they choose. If the user doesn't set one, then we either need to check for null on every use or create a dummy one during initialize (could be an empty composite underneath).

Reply:
I agree.  @sconde, do you have a use case that would require a Stepper have a composite observer?

Reply:
Ok, I have pushed on to this branch changes that remove composite observers from the Steppers and make them just observers.  This really cleaned up some code, so I think it is for the better.